README.md

```diff
@@ -50,12 +50,12 @@ module.exports = {
 
 ## Options
 
-|                Name                 |        Type         |                   Default                    | Description                                      |
-| :---------------------------------: | :-----------------: | :------------------------------------------: | :----------------------------------------------- |
-|   **[`attributes`](#attributes)**   | `{Boolean\|Object}` |                    `true`                    | Enables/Disables attributes handling             |
+|                Name                 |        Type         |                   Default                    |                   Description                    |
+| :---------------------------------: | :-----------------: | :------------------------------------------: | :----------------------------------------------: |
+|   **[`attributes`](#attributes)**   | `{Boolean\|Object}` |                    `true`                    |       Enables/Disables attributes handling       |
 | **[`preprocessor`](#preprocessor)** |    `{Function}`     |                 `undefined`                  | Allows pre-processing of content before handling |
-|     **[`minimize`](#minimize)**     | `{Boolean\|Object}` | `true` in production mode, otherwise `false` | Tell `html-loader` to minimize HTML              |
-|     **[`esModule`](#esmodule)**     |     `{Boolean}`     |                   `false`                    | Use ES modules syntax                            |
+|     **[`minimize`](#minimize)**     | `{Boolean\|Object}` | `true` in production mode, otherwise `false` |       Tell `html-loader` to minimize HTML        |
+|     **[`esModule`](#esmodule)**     |     `{Boolean}`     |                   `false`                    |              Use ES modules syntax               |
```

Comment:
Why it is changed, if not new?

Reply:
Oh I added `:` in the last of --- under the description table which made the lint centralize the description 😅